### ISSUE TITLE
Handle standard CRUD functions

### ISSUE THREAD

**Issue by hashwin** [lexich/redux-api#59]
Hi,

First of all, great plugin, its working really well for us.
It would be nice if it could handle standard CRUD functions for a standard REST API that you could override as necessary. I currently have it set up like this: 

```
const URL = 'https://example.com/api/v1'
function resourceOptions(resourceName) {
  return {
    url: `${URL}/${pluralize(resourceName)}/:id`,
    helpers: {
      update(id, resource) {
        const body = {};
        body[resourceName] = resource;
        const urlParams = { id };
        const params = {
          method: 'patch',
          body: JSON.stringify(body),
          headers: headers
        };
        return [urlParams, params];
      },

      create(resource) {
        const body = {};
        body[resourceName] = resource;
        const urlParams = {};
        const params = {
          method: 'post',
          body: JSON.stringify(body),
          headers: headers
        };
        return [urlParams, params];
      },

      delete(id) {
        const urlParams = { id };
        const params = {
          method: 'delete',
          headers: headers
        };
        return [urlParams, params];
      }
    }
  };
}

var options = {};
['test', 'post'].forEach(resource => {
  var plural = pluralize(resource);
  options[resource] = resourceOptions(resource);
  options[plural] = indexOptions(plural);
});

export default reduxApi(options)
```
This generates the rest API functions for tests and posts.
Adding a function like this into the plugin would mean even less boilerplate code :smile: 
All you would need then is a base URL, common headers, and an array of endpoint resource names.

**Comment by lexich** [comment]
I think that It's not bad idea.

**Comment by lexich** [comment]
@hashwin what do you think about this PR #60

**Comment by lexich** [comment]
closed with #60
